client/python/tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 isolated_build = true
 envlist =
-    format
+    format-code
```

Comment:
I think its bad practive to auto format code in a commit.  

We have format because we want the committer to run the formatter and commit his changes.  

It gets nasty if you have git auto format because then it should also commit its changes as a separate commit.  So let's avoid that.

Reply:
Oops this was not meant to be commited. Thanks for spotting :)